server/backend/api/app/routers/router_guards.py

```diff
@@ -273,3 +370,84 @@ async def enforce_self_grant_guard(
                 },
                 headers={"WWW-Authenticate": "Bearer"},
             )
+
+
+async def enforce_bceid_by_same_org_guard(
+    request: Request,
+    # forbid business bceid user (requester) manage idir user's access
+    _enforce_user_type_auth: None = Depends(authorize_by_user_type),
```

Comment:
if for the case business bceid user manage idir user access, will raise exception here. So for this call, it will only check the case when business bceid manage business bceid